tests/build_test.sh

```diff
@@ -0,0 +1,122 @@
+include './src/build.sh'
+include './tests/utils.sh'
+
+function get_kernel_release_mock()
+{
+  echo '5.4.0-rc7-test'
+}
+
+function get_kernel_version_mock()
+{
+  echo '5.4.0-rc7'
+}
+
+oneTimeSetUp()
```

Comment:
Why you did not add a function? In the original function, we had it. Same for setUp() and tearDown()

Reply:
@rodrigosiqueira, I didn't understand your question. What function was not added?

Reply:
I think he means in the fucntion definitions, we usually do something like `function foo(){ stuff}`, but the keyword `function`is missing here (nice catch)

Reply:
Yes, I was talking about the keyword `function`. Anyway, I reviewed this patch again, and I decided to not waste reviewing cycles and fix this and other minor issues by myself.